specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/virtualWan.json

```diff
@@ -34,7 +34,7 @@
     }
   },
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans/{VirtualWANName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans/{virtualWanName}": {
```

Comment:
Please update all the parameter definition under this path.  I only see the definition in 'get' is updated, there are also 'put', 'delete' call which still use the original name.

Reply:
Updated, thanks for a nice catch!